core/templates/domain/editor/undo_redo/base-undo-redo.service.ts

```diff
@@ -101,7 +101,7 @@ export class BaseUndoRedo {
    * returned list will not be reflected in this class instance.
    */
   getChangeList(): Change[] {
-    // TODO(bhenning): Consider integrating something like Immutable.js to
+    // TODO(#20337): Consider integrating something like Immutable.js to
```

Comment:
Thank you @FaithAbiola for creating the PR, the issue titles (for this and others) can be rephrased? Please remove the filepath from the title of the Issue and mention the filepath as a part of the description.
Thank you

Reply:
Hi @Nik-09, thank you for the feedback. I will do that now.

Reply:
Hi @Nik-09,

I have made the changes. Kindly review them.

Thank you.

Reply:
Hi @FaithAbiola thank you for updating the issue descriptions. 
_In general, a reviewer should mark a conversation resolved._

Reply:
Hi @Nik-09, 

Thank you for bringing this to my attention. That was a mistake on my part. 

I'll be more careful in the future to avoid this.